AFQ/viz/utils.py

```diff
@@ -368,6 +368,9 @@ def __init__(self,
             self.visualize_roi = AFQ.viz.plotly_backend.visualize_roi
             self.visualize_volume = AFQ.viz.plotly_backend.visualize_volume
             self.create_gif = AFQ.viz.plotly_backend.create_gif
+        else:
+            TypeError("Visualization backend should be"
+                      + " either 'plotly' or 'fury'")
```

Comment:
Always a good idea to tell them what the value was at that moment. In case the user wasn't the one setting the value themselves.

```suggestion
                      + " either 'plotly' or 'fury'. It is currently set to %s"%self.viz.backend)
```